### PR TITLE
ci: Travis: dist=xenial

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
+dist: xenial
```
Comment (tiangolo) [inline .travis.yml]: Why Xenial? What for?

Comment (blueyed) [reply .travis.yml]: It is newer, i.e. upgraded packages etc.
And it provides `3.7` out of the box, instead of 3.7-dev.

Comment (tiangolo) [reply .travis.yml]: Ah! I get it now. I was assuming Travis was using a newer version.

**Comment by tiangolo** [comment]
Great. Thanks! :tada: :taco:

**Pull request merged**